### DISCUSSION TITLE
Nuxt3 and Store

### DISCUSSION THREAD

**Discussion by aslero** [nuxt/nuxt#18295] [Questions]
Good evening, how do I use the Store in Nuxt3?

**Comment by Luffyyy** [comment]
nuxt/nuxt#16994
At the moment, the closest you have is https://v3.nuxtjs.org/docs/usage/state

**Reply by kakajansh** [reply]
Worth considering [pinia](https://pinia.esm.dev/). Nuxt3 ready
Sample app [nuxt3-app](https://github.com/productdevbook/nuxt3-app)

**Reply by Luffyyy** [reply]
Yeah it's nice
